src/cli/expand-patterns.js

```diff
@@ -20,15 +20,15 @@ async function* expandPatterns(context) {
       continue;
     }
 
-    const fileName = path.resolve(filePath);
+    const filename = path.resolve(filePath);
```

Comment:
FYI: I'm waiting for fast-glob v4 to use `absolute: true`.  There are bugs in v3. https://github.com/mrmlnc/fast-glob/issues/371